index.html

```diff
@@ -267,12 +267,20 @@
   <body>
     <section id='abstract'>
       <p>
-[=Credentials=] are integral to our daily lives; driver's licenses confirm
-our capability to operate motor vehicles, university degrees assert our level
-of education, and government-issued passports permit travel between countries.
-This specification provides a mechanism to express these sorts of
-[=credentials=] on the Web in cryptographically secure, privacy-respecting,
-and machine-verifiable way.
+This specification provides normative requirements and non-normative guidance
+for expressing and consuming [=verifiable credentials=] and associated
+cryptographic practices. It emphasizes the importance of understanding and
+updating cryptographic systems and managing private signing keys with limited
+validity periods. It discusses mechanisms for ensuring content integrity of
+linked external resources and highlights the risks of unsigned claims.
+Strategies are provided to mitigate man-in-the-middle (MITM), replay, and
+spoofing attacks, and to address issues related to credential atomization,
+validity periods, and device security. This specification also covers
+acceptable use of credentials, warns against code injection risks, and
+underscores the need for accessibility and internationalization
+considerations, advocating for a data-first approach and adherence to
+internationalization standards to ensure correct rendering of
+multilingual text.
```

Comment:
```suggestion
A [=verifiable credential=] is a digital representation of a set of [=claims=] made 
by an [=issuer=], such as a driver's license or a education certificate. This 
specification describes an extensible data model for [=verifiable credentials=], 
how they can be secured from tampering, and a three-party ecosystem that 
exchanges these credentials that is composed of [=issuers=], [=holders=], 
and [=verifiers=]. This document also covers a variety of security, privacy,
internationalization and accessibility considerations for ecosystems that 
use the technologies described in this specification.
```

Reply:
Here is a proposal for an abstract rewrite that I would be a +1 to merge (with @TallTed's inevitable change suggestions that improve the text).

Reply:
I believe the above suggestion should say "A [=verifiable credential=] is a specific way to express a set of [=claims=]". VCs can be printed / embedded physically and even if the information itself remains digital, this might confuse someone in an abstract.

Reply:
@dlongley, applied in the change suggestion above.

Reply:
Please reword the following
"and a three-party ecosystem that  exchanges these credentials that is composed of [=issuers=], [=holders=], 
and [=verifiers=]. "
to
"and a three-party ecosystem for the exchange of these credentials that is composed of [=issuers=], [=holders=], 
and [=verifiers=]. "